contrib/scripts/check-fmt.sh

```diff
@@ -1,13 +1,24 @@
-#!/usr/bin/env sh
+#!/usr/bin/env bash
 
 set -e
-diff="$(find . ! \( -path './contrib' -prune \) \
+set -o pipefail
+
+if find ./daemon/bindata.go ; then
+  diff="$(find . ! \( -path './contrib' -prune \) \
         ! \( -path './vendor' -prune \) \
         ! \( -path './.git' -prune \) \
         ! \( -path '*.validate.go' -prune \) \
         ! -samefile ./daemon/bindata.go \
         -type f -name '*.go' -print0 \
                 | xargs -0 gofmt -d -l -s )"
+else
```

Comment:
Maybe could use `find ... | grep -v "daemon/bindata.go" | xargs ...` to not require duplicating the code like this?

Reply:
will try that